config/_default_shared.yml

```diff
@@ -210,43 +210,6 @@ Naming/FileName:
 Naming/MethodName:
   Enabled: true
 
-Performance/CaseWhenSplat:
```

Comment:
do we need to update readme to document this break change?

Reply:
Yeah. I discussed with @joelhawksley and we decided instead to require the performance gem:

https://github.com/github/rubocop-github/pull/63